terraform/database.tf

```diff
@@ -285,6 +285,95 @@ resource "null_resource" "migrate" {
   ]
 }
 
+# Create a storage bucket where database backups will be housed.
+resource "google_storage_bucket" "backups" {
+  project  = var.project
+  name     = "${var.project}-backups"
+  location = var.storage_location
+
+  force_destroy               = true
```

Comment:
feels safer to me to have this be false(the default).

Reply:
This tells terraform to delete the objects in the bucket before deleting the bucket. GCS doesn't let you delete a bucket with objects inside.

Reply:
Yeah, I understood that. I felt it was kinda safer to require the bucket be empty.